server/routes/link/requestLink.spec.ts

```diff
@@ -0,0 +1,87 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import nock from 'nock'
+import config from '../../config'
+import mockHmppsAuth from '../fixtures/mock-hmpps-auth'
+import assertThat from '../fixtures/supertest-assertions'
+import SuperTestWrapper from '../fixtures/SuperTestWrapper'
+
+describe('Request Link', () => {
+  const superTest = new SuperTestWrapper()
+  let mockedSendLegalMailApi: nock.Scope
+
+  beforeEach(() => {
+    mockHmppsAuth()
+    mockedSendLegalMailApi = nock(config.apis.sendLegalMail.url)
+  })
+
+  afterEach(() => {
+    nock.cleanAll()
+  })
+
+  it('should render page to request a link', async () => {
+    const response = await superTest.request //
```

Comment:
Does the `////` at the end mean anything?

Reply:
No, it's just a way I can shoehorn my desire to lay out the code into a more readable manner (IMHO) rather than being at the whim of the draconian prettier rules 😁 
To be fair this particular instance is not that offensive because there is only one method call being chained:
```
const response = await superTest.request //
      .get('/link/request-link')
```
Without the comment markers prettier would force me to do:
```
const response = await superTest.request.get('/link/request-link')
```

That's no _so_ bad, but there are other cases below where I chain more method calls:
```
const response = await superTest.request //
      .post('/link/request-link')
      .send({ email: 'user@aardvark.com.cjsm.net' })
```
If prettier had it's way it would force me to do:
```
const response = await superTest.request.post('/link/request-link').send({ email: 'user@aardvark.com.cjsm.net' })
```
Horrible 🤮